Fix for Linux int driver omitting encryption

akutz opened the pull request:
This patch updates the Linux integration driver so that it processes volume creation requests with encryption properly.

codenrhoden commented on drivers/integration/linux/linux.go:
```diff
@@ -459,8 +459,10 @@ func (d *driver) Create(
 	volumeType := d.volumeType()
 	optsNew.Type = &volumeType
 	io, _ := strconv.Atoi(d.iops())
```
I know it's not part of this PR, but I hate seeing this err suppressed. It doesn't appear that the syntax for this config value has been validated previously, so it's possible this conversion should fail. Something worth addressing?

akutz replied:
Hi @codenrhoden,

I addressed this concern in a rebase. Care to review?

akutz committed:
Fix for Linux int driver omitting encryption

This patch updates the Linux integration driver so that it processes
volume creation requests with encryption properly.
